src/components/Button/Button.js

```diff
@@ -44,7 +49,17 @@ const Button = props => {
     className,
     title,
     style,
+    ariaLabel,
   } = { ...props, ...customOverrides };
+  const [t] = useTranslation();
+
+  let aLabel = ariaLabel;
+  if (!ariaLabel && title) {
+    aLabel = t(title);
+  }
```

Comment:
Nevermind this is better and still readable:

```suggestion
  const aLabel = ariaLabel || title ? t(title) : undefined;
```